packages/babel-plugin-strip-runtime/src/index.tsx

```diff
@@ -13,7 +13,17 @@ export default declare<PluginPass>((api) => {
 
   return {
     name: '@compiled/babel-plugin-strip-runtime',
+    pre() {
+      this.styleRules = [];
+    },
     visitor: {
+      Program: {
```

Comment:
modified this to be a little less dumb -- now it behaves the same as the primary babel plugin. on _completion_ of the pass, callback. instead of multiple times.